flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "skip-appstream-check": true
```

Comment:
Can the file be fixed instead?

Reply:
I can't make that decision. If you read above, I'm skipping this because Alexander licensed the appstream metadata under GPL2.0+ which is invalid according to appstream-glib. I've asked them to relicense it to fix this issue but they've not responded in over 3weeks therefore I decided to skip it until I can get an answer. See #6 and the comment above.